ionoscloud/resource_server_test.go

```diff
@@ -1868,8 +1900,47 @@ resource ` + constant.ServerResource + ` ` + constant.ServerTestResource + ` {
     firewall_active = false
   }
 }`
+const testAccCheckServerConfigNoVolume = `
```

Comment:
@cristiGuranIonos I would change this to `testAccCheckServerNoInlineVolumeConfig`, I would emphasize the fact that the volume is not inline (we have an external volume in this configuration).